1-exercises/E-while-loop-with-array/exercise.js

```diff
@@ -5,19 +5,26 @@
 */
 
 const BIRTHDAYS = [
-    "January 7th",
-    "February 12th",
-    "April 3rd",
-    "April 5th",
-    "May 3rd",
-    "July 11th",
-    "July 17th",
-    "September 28th",
-    "November 15th"
+  { b_month: "January", b_date: "7th" },
+  { b_month: "February", b_date: "12th" },
+  { b_month: "April", b_date: "3rd" },
+  { b_month: "April", b_date: "5th" },
+  { b_month: "May", b_date: "3rd" },
+  { b_month: "July", b_date: "11th" },
+  { b_month: "July", b_date: "17th" },
+  { b_month: "September", b_date: "28th" },
+  { b_month: "November", b_date: "15th" },
 ];
 
+let birthdays = [];
 function findFirstJulyBDay(birthdays) {
-    // TODO
+  let month = birthdays.find(function (month) {
+    return month.b_month.indexOf("July") > -1;
+  });
+  return `${month.b_month} ${month.b_date}`;
```

Comment:
This will work but it's kind of cheating since you changed the array given 😆
Could you do this using Regex for example?

Reply:
lols I'll try Regex then. Thanks for the suggestion.